### PR TITLE
fix log level 0

### PR DESCRIPTION
**Pull request by MegaGM**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -14,8 +14,17 @@ var STATE_SERVER_ACK_TIMEOUT = Number(process.env.SCC_STATE_SERVER_ACK_TIMEOUT)
...
+/**
+ * Log levels:
+ * 3 - log everything
+ * 2 - warnings and errors
+ * 1 - errors only
+ * 0 - log nothing
+ */
+var LOG_LEVEL = (typeof argv.l !== 'undefined') ?
+  Number(argv.l) : (typeof process.env.SCC_BROKER_SERVER_LOG_LEVEL !== 'undefined') ?
+  Number(process.env.SCC_BROKER_SERVER_LOG_LEVEL) : 1;
```
Comment (jondubois) [inline server.js]: We should use `if-else` statement here. The double `?:?:` notation is a bit confusing for people like me ;p

File: server.js [hunk 2]
```diff
@@ -73,7 +82,7 @@ var connectToClusterStateServer = function () {
   var stateSocket = scClient.connect(scStateSocketOptions);
 
   stateSocket.on('error', (err) => {
-    console.error(err);
+    (LOG_LEVEL > 0) && console.error(err);
```
Comment (jondubois) [inline server.js]: Optional (depends on what you decide in scc-state PR), but maybe consider using `if` statement here? Consistency is good.
 The rest of SC uses `if` for doing comparison checks. See https://github.com/SocketCluster/socketcluster/blob/master/index.js#L568

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "scc-broker",
-  "version": "1.6.4",
+  "version": "1.6.6",
```
Comment (jondubois) [inline package.json]: Like scc-state PR, no need to bump the version number between each commit; just once at the end ;p
Also I only bump up when I'm about to do a release so you could just let me do it.

**Review by jondubois — CHANGES_REQUESTED** [review]
Nice work. I added some feedback.

File: package.json (hunk 3 above)
Comment (MegaGM) [reply package.json]: Sorry for the version bumps. I won't do them anymore. Apparently I have `npm version patch` somewhere beneath my `git add --force commit --force push --force npm publish --force` goddamn shell alias :joy_cat:

**Pull request merged**